string id
Mesh mesh
Area[] areas
